### PR TITLE
Add hide donation message option in add subcommand

### PR DESCRIPTION
**Pull request by atamakahere-git**
Enhancement for #369

### CODE REVIEW

**Comment by dr-vortex** [comment]
@atamakahere-git Any updates on this?

File: src/cli.rs [hunk 1]
```diff
@@ -51,6 +51,10 @@ pub enum SubCommands {
         /// The mod loader will not be checked for this mod
         #[clap(long, short = 'M', alias = "dont-check-mod-loader")]
         ignore_mod_loader: bool,
+
+        /// Ignore the donation message
+        #[clap(long, short = 'H', alias = "hide-donation-message")]
```
Comment (theRookieCoder) [inline src/cli.rs:56]: The alias here is unnecessary, clap will automatically convert Rust snake_case variables and PascalCase subcommands to the standard kebab-case used in CLIs.

```diff
- #[clap(long, short = 'H', alias = "hide-donation-message")]
+ #[clap(long, short = 'H')]
```

**Review by theRookieCoder — CHANGES_REQUESTED** [review]

File: src/cli.rs (hunk 1 above)
Comment (theRookieCoder) [reply src/cli.rs:56]: Also I think `Hide the donation message` is good for the description too

**Comment by theRookieCoder** [comment]
Donation messages would've been quite difficult to deal with when adding multiple mods, so I ended up removing them entirely.

**Pull request closed without merging**